bot/cogs/memes_cog.py

```diff
@@ -3,6 +3,12 @@
 import discord
 import discord.ext.commands as commands
 
+from random import randint
+from random import seed
```

Comment:
make this just `import random` then do `random.randint` or `random.seed`

you could also alias that to like r or rand or something

Reply:
I did but I figured it'd be more memory effecient to import just one function rather than an entire module

Reply:
that wont matter, `from random import randint` still imports the entire module, it just only exposes randint. you arent saving memory either way